hypervisor/src/vm.rs

```diff
@@ -404,6 +410,11 @@ pub trait Vm: Send + Sync + Any {
     fn resume(&self) -> Result<()> {
         Ok(())
     }
+
+    #[cfg(feature = "sev_snp")]
+    fn gain_page_access(&self, _gpa: u64, _size: u32) -> Result<()> {
+        unimplemented!()
```

Comment:
The default implementation can use `Ok(())` here. You don't need to have a hook for KVM.